code/datums/uplink_items/uplink_general.dm

```diff
@@ -948,6 +948,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/autosurgeon/organ/syndicate/oneuse/scope_eyes
 	cost = 10
 
+/datum/uplink_item/cyber_implants/mantis_kit
+	name = "'Naginata' Mantis Blades Kit"
+	desc = "A pair of devastating 'Naginata' concealable mantis blades, which retract into the arms of the user. \
+	Their monomolecular edges will easily tear through flesh and armor alike, and can even pry open airlocks when used together. \
```

Comment:
I think you should probably include in some in-game item description somewhere (either in the uplink, the examine, and/or in flavor text when first implanted) that note that you need to have both equipped in order to get the full benefits of them.